_collections/_section_android/optionalSdkParams.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Optional SDK Properties
+position: 5
+type: Android
+description:
+right_code: >-
+---
+
+The SDK has some optional properties that can be passed in initialisation to use and/or display certain features
+
+  ```kotlin
+ CartrawlerSDK.Builder()
+            .enableCustomCashTreatment()
+            .setUSPDisplayType(USPDisplayType.CHECK_STYLE)
+  ```
+
+<dl>
+<dt>enableCustomCashTreatment</dt>
+<dd>This used in the SDK as a main toggle to display some features related to the Cash Special Offer</dd>
```

Comment:
This is used in the SDK in order to enable enhanced cash voucher merchandising throughout the booking flow. (Cash voucher banner, Cash voucher widgets etc)